debezium-storage/debezium-storage-rocketmq/src/main/java/io/debezium/storage/rocketmq/history/RocketMqSchemaHistory.java

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.storage.rocketmq.history;
+
+import io.debezium.annotation.NotThreadSafe;
+import io.debezium.config.Configuration;
+import io.debezium.config.Field;
+import io.debezium.document.DocumentReader;
+import io.debezium.relational.HistorizedRelationalDatabaseConnectorConfig;
+import io.debezium.relational.history.AbstractSchemaHistory;
+import io.debezium.relational.history.HistoryRecord;
+import io.debezium.relational.history.HistoryRecordComparator;
+import io.debezium.relational.history.SchemaHistory;
+import io.debezium.relational.history.SchemaHistoryException;
+import io.debezium.relational.history.SchemaHistoryListener;
+import io.debezium.storage.rocketmq.RocketMqAdminUtil;
+import io.debezium.storage.rocketmq.RocketMqConfig;
+import io.debezium.storage.rocketmq.ZeroMessageQueueSelector;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.rocketmq.client.consumer.DefaultLitePullConsumer;
+import org.apache.rocketmq.client.exception.MQBrokerException;
+import org.apache.rocketmq.client.exception.MQClientException;
+import org.apache.rocketmq.client.producer.DefaultMQProducer;
+import org.apache.rocketmq.client.producer.SendCallback;
+import org.apache.rocketmq.client.producer.SendResult;
+import org.apache.rocketmq.common.TopicConfig;
+import org.apache.rocketmq.common.admin.TopicOffset;
+import org.apache.rocketmq.common.message.Message;
+import org.apache.rocketmq.common.message.MessageExt;
+import org.apache.rocketmq.common.message.MessageQueue;
+import org.apache.rocketmq.remoting.exception.RemotingException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.function.Consumer;
+
+/**
+ * @author sunxiaojian
+ */
+@NotThreadSafe
+public class RocketMqSchemaHistory extends AbstractSchemaHistory {
+    private final static Logger LOGGER = LoggerFactory.getLogger(RocketMqSchemaHistory.class);
+    private static final int UNLIMITED_VALUE = -1;
```

Comment:
I'd assume that `UNLIMITED_VALUE` will contain something like infinity, not `-1`. Maybe just use it directly in the code `long lastProcessedOffset = -1`, it's IMHO more understandable when `long lastProcessedOffset = UNLIMITED_VALUE;` (and yes, I noticed that it's used also in `KafkaSchemaHistory`, but it doesn't mean that I like it, it's already there:-()